src/components/CreateGroupDialog.vue

```diff
@@ -58,6 +70,22 @@ export default {
             this.$emit("added", this.groupName);
             this.modal.hide();
         },
+        /**
+         * Clean up modal and restore scroll behavior
+         * @returns {void}
+         */
+        cleanupModal() {
+            if (this.modal) {
+                try {
+                    this.modal.hide();
+                } catch (e) {
+                    console.warn("Modal hide failed:", e);
+                }
+            }
+            document.body.classList.remove("modal-open");
+            document.body.style.paddingRight = "";
+            document.body.style.overflow = "";
```

Comment:
as far as I see these are already removed by `this.modal.hide()`.
Is there a case where you needed them which I am missing?

```suggestion
```

Reply:
nope it wasnt actually removing those details, so thought to manually remove them

Reply:
nvm now tht i removed them still its working properly now too :)